src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -116,6 +118,8 @@
   margin-inline-end: var(--#{$breadcrumb}__menu-toggle--MarginInlineEnd);
 
   .#{$menu-toggle} {
+    padding-block-start: var(--#{$breadcrumb}__menu-toggle--PaddingBlockStart);
+    padding-block-end: var(--#{$breadcrumb}__menu-toggle--PaddingBlockEnd);
     line-height: var(--#{$breadcrumb}__menu-toggle--c-menu-toggle--LineHeight);
   }
```

Comment:
discussed in the working section that we can keep the padding as long as it it doesn't change the shape of the outer breadcrumb box. also looks like `line-height` is the same with or without this declaration, so I think we can nix this whole block!